Tools/autotest/default_params/sub.parm

```diff
@@ -58,9 +58,10 @@ INS_ACC3OFFS_Z   0.000
 INS_ACC3SCAL_X   1.000
 INS_ACC3SCAL_Y   1.000
 INS_ACC3SCAL_Z   1.000
-PSC_ACCZ_P   4
-PSC_VECZ_P   8
-PSC_POSZ_P   3
+PSC_POSXY_P 2.5
```

Comment:
why are we replacing z with xy, instead of appending?

Reply:
oops